flink-dist/src/main/flink-bin/bin/config.sh

```diff
@@ -99,6 +99,9 @@ DEFAULT_ENV_JAVA_OPTS_HS=""                         # Optional JVM args (History
 DEFAULT_ENV_SSH_OPTS=""                             # Optional SSH parameters running in cluster mode
 DEFAULT_YARN_CONF_DIR=""                            # YARN Configuration Directory, if necessary
 DEFAULT_HADOOP_CONF_DIR=""                          # Hadoop Configuration Directory, if necessary
+DEFAULT_JVM_HEAPDUMP_ON_OOM="true"                  # Whether enable heap dump on ooms
+DEFAULT_JVM_HEAPDUMP_DIRECTORY="/tmp"               # Heap dump directory
```

Comment:
Do we need to add a new directory here? Can we reuse some directory already used in Flink?

Reply:
This directory actually already exists as the DEFAULT_ENV_PID_DIR, please check config.sh#L93